src/definition/rooms/IRoomUserTyping.ts

```diff
@@ -10,5 +10,11 @@ import { IRoomUserTypingContext } from './IRoomUserTypingContext';
  *
  */
 export interface IRoomUserTyping {
-    executeOnRoomUserTyping(context: IRoomUserTypingContext, read: IRead, http: IHttp, persistence: IPersistence): Promise<void>;
+    executeOnRoomUserTyping(
```

Comment:
@csmith14 `yarn lint` is giving some errors, Can you please check? Also after all the changes are ready, can you run `yarn compile` to update build files as a separate commit?